### ISSUE TITLE
Keep the latest pasted item at top of history

### ISSUE THREAD

**Issue by vcarel** [aefernandes/vscode-copypaste-history-extension#5]
When I'm using an item from the history, it stays at its place.

It annoys me because some items I frequently use get farther and father in the history. At some time, I have to copy them again so to get them on the top again.

I think it would make sense to move to the top an item when it's pasted.

BTW this is what happens with [atom-clipboard-plus](https://github.com/aki77/atom-clipboard-plus).

**Comment by kercos** [comment]
Indeed, I agree with @vcarel that this feature would be very useful!

**Comment by heldersepu** [comment]
Very nice indeed! go LRU

**Comment by s1moe2** [comment]
This was also something I needed, specially since I was used to this feature in IntelliJ IDE's.
I've submitted [this PR](https://github.com/aefernandes/vscode-clipboard-history-extension/pull/12) which handles that and also has some other minor fixes/cleanup.

**Comment by kAlvaro** [comment]
One more reason for such feature is that, otherwise, frequently used entries can easily disappear as new items are being copied, when history limit is hit.

_BTW, there're already two different pull requests to implement this :-?_

**Comment by heldersepu** [comment]
I don't think the owner (@aefernandes) is getting notifications from the issues...
try reaching out on linkedin: https://www.linkedin.com/in/anjalifernandes/

**Comment by s1moe2** [comment]
I've sent her an email a couple of weeks ago offering to help as maintainer but got no response. I'm now considering doing the changes I want to do (this issue and other things) on my fork and using it instead of the original.

@kAlvaro that's true! I guess I got excited and started working without noticing the existing PR :) I also added irrelevant refactors, which maybe should have been kept out but I also made this behavior optional/configurable
